egs/sprakbanken/s5/conf/online_cmvn.conf

```diff
@@ -0,0 +1 @@
+# configuration file for apply-cmvn-online, used in the script ../local/run_online_decoding.sh
```

Comment:
`local/run_online_decoding.sh` does not exist for this recipe. Maybe you accidentally copied this file from tedlium.